contrib/patterns/_grid-push.scss

```diff
@@ -1,7 +1,7 @@
-.grid-push--3 {
-  @include grid-push(3);
+.grid-shift--4 {
```

Comment:
Should this partial now be called `_grid-shift.scss` instead of `_grid-push.scss`?